### PR TITLE
SW-4383 List Inventory by Nursery

### PR DESCRIPTION
**Pull request by constanzauanini**
Use same InventoryTable from inventoryBySpecies but change columns and filter results according to what is needed

Screenshot 2023-11-17 at 10 17 04

### CODE REVIEW

File: src/components/InventoryV2/InventoryListByNursery.tsx [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+    const nextResults = apiSearchResults?.reduce((acc, result) => {
+      const resultTyped = result as FacilityInventoryResult;
+      const indexFound = acc.findIndex((savedResult) => savedResult.facility_id === resultTyped.facility_id);
+
+      if (indexFound !== undefined && indexFound !== -1) {
+        const existingFacility = acc[indexFound];
+        acc[indexFound] = {
+          ...existingFacility,
+          germinatingQuantity: (
+            Number(existingFacility.germinatingQuantity) + Number(resultTyped['germinatingQuantity(raw)'])
```
Comment (nickgraz) [inline src/components/InventoryV2/InventoryListByNursery.tsx:105]: Are these safe to do math on? `existingFacility.germinatingQuantity`

File: src/components/InventoryV2/InventoryListByNursery.tsx [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+      searchSortOrder,
+    });
+
+    let updatedResult: InventoryResultWithSpeciesNames[] | undefined = [];
+
+    const nextResults = apiSearchResults?.reduce((acc, result) => {
+      const resultTyped = result as FacilityInventoryResult;
+      const indexFound = acc.findIndex((savedResult) => savedResult.facility_id === resultTyped.facility_id);
+
+      if (indexFound !== undefined && indexFound !== -1) {
```
Comment (nickgraz) [inline src/components/InventoryV2/InventoryListByNursery.tsx:100]: This can also be `if (indexFound > -1) {`. Works as expected for `undefined` and `-1`

**Review by nickgraz — APPROVED** [review]
Looks good to me! One small question that is probably inconsequential

File: src/components/InventoryV2/InventoryListByNursery.tsx (hunk 1 above)
Comment (constanzauanini) [reply src/components/InventoryV2/InventoryListByNursery.tsx:105]: This page is only shown for nursery2, so I guess it is

**Pull request merged**